Procfile

```diff
@@ -2,9 +2,9 @@
 #
 # Start/spawn all processes with `honcho start` at the command-line.
 # Stop children processes with Control+C to send the interrupt signal.
-# 
+#
 # For more information, see: https://honcho.readthedocs.io/en/latest/index.html#what-are-procfiles
 
 proxy: caddy run
 backend: uvicorn --port=1561 --reload backend.main:app
-frontend: cd frontend && ng serve --poll 2000
\ No newline at end of file
+frontend: cd frontend && if [[ "$OSTYPE" =~ ^msys ]]; then ng serve --poll 2000; else ng serve; fi
```

Comment:
We may ultimately have to do some kind of `if`, like you attempted, on the contents of an env var. It looks like if you set `poll` to 0, it will default to 500, which is not what we would want: https://github.com/angular/angular-cli/blob/main/packages/angular_devkit/build_angular/src/utils/normalize-builder-schema.ts#L72-L75

Reply:
What if we just made the variable $HOST_OS in our `.env` file and asked Windows users to report `windows`, macOS users to report `macos`, then we test for `^win` in this conditional? This seems like the most straightforward way to get students to setup the .env file with one extra step and may be useful in other places where file sync issues are concerned.